### DISCUSSION TITLE
Unable to access local lan

### DISCUSSION THREAD

**Discussion by sharkaccident** [wg-easy/wg-easy#303] [Q&A] [answered]
My local lan is 192.168.0.x . I am trying to use wg-easy to access server 192.168.0.15 from outside my network. I am unable to see any 192.168.0.x computers when I am connect to wg-easy.

Here is docker compose file:
`
version: "3.8"
services:
  wg-easy:
    environment:
      # ⚠️ Required:
      # Change this to your host's public address
      -  WG_HOST=vpn.super_secret.com
     
      # Optional:
      - PASSWORD=pw
      # - WG_PORT=51820
      # - WG_DEFAULT_ADDRESS=10.8.0.x
      - WG_DEFAULT_DNS=1.1.1.1
      # - WG_MTU=1420
      - WG_ALLOWED_IPS=192.168.0.0/24, 10.8.0.0/24
      # - WG_PRE_UP=echo "Pre Up" > /etc/wireguard/pre-up.txt
      # - WG_POST_UP=echo "Post Up" > /etc/wireguard/post-up.txt
      # - WG_PRE_DOWN=echo "Pre Down" > /etc/wireguard/pre-down.txt
      # - WG_POST_DOWN=echo "Post Down" > /etc/wireguard/post-down.txt
      
    image: weejewel/wg-easy
    container_name: wg-easy
    volumes:
      - /mnt/qnap/wg_easy/config:/etc/wireguard
    ports:
      - "51820:51820/udp"
      - "51821:51821/tcp"
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
      - SYS_MODULE
    sysctls:
      - net.ipv4.ip_forward=1
      - net.ipv4.conf.all.src_valid_mark=1`

I have port forwarded 51820 to wg-easy server on my local lan. I also have cloudflare directing vpn.super_secret.com to my router. I am able to establish link between peer and wg-easy but when I try to connect to local ip addresses (192.168.0.x) I time out on peer. 

If I log into docker console I am able to ping 192.168.0.15 and www.google.com

**Comment by ferby09** [comment]
Got the same problem. Any help?
I'm using the docker in unRAID. Could it be that the variable WG_ALLOWED_IPS is ignored?
Because the client still says allowed iOS 0.0.0.0/0, ::/0

**Reply by pandinusimp** [reply, marked as answer]
I would not say ignored. 

It creates the same IP Adress for both server and client. Thats i think its the problem.
Take a look at wg0.conf and wg0.json.

That's is WG-Easy with a custom IP adress. you can see the IP adresses of server and client is the same.

![WG JSON Custom IP](https://user-images.githubusercontent.com/112988214/188751328-d89fd3c6-8f43-41aa-82ee-af888ac225df.png)

That's is WG-Easy with the default setting. The IP adresses are different between server and client.

![WG JSON Default](https://user-images.githubusercontent.com/112988214/188751691-91338850-cd5d-488e-9471-3e148a965dae.png)

**Reply by ferby09** [reply]
Thanks for your reply. I was just too stupid to find the build-in Wireguard in unRAID. Using this works now.
